app/components/Notification/index.jsx

```diff
@@ -129,7 +145,7 @@ let Notification = React.createClass({
                                   iconClassName={iconClassName}/>;
         let menu = this.state.notifications.map(n => {
             let sender = UserStore.getUser(n.from_user_id);
-            return <NotifiItem key={n.id} sender={sender} time={n.created_at} message={n.content} read={n.checked}/>;
+            return <NotifiItem id={n.id} url={n.url} sender={sender} time={n.created_at} message={n.content} read={n.checked}/>;
```

Comment:
Don't remove `key` property, it is required by React.
https://facebook.github.io/react/docs/create-fragment.html